lib/ts/fetch.ts

```diff
@@ -304,19 +345,6 @@ export default class AuthHttpRequest {
                     if (response.status === AuthHttpRequest.config.invalidClaimStatusCode) {
                         await onInvalidClaimResponse(response);
                     }
-                    const antiCsrfToken = response.headers.get("anti-csrf");
-                    if (antiCsrfToken) {
-                        const tok = await getIdRefreshToken(true);
-                        if (tok.status === "EXISTS") {
-                            logDebugMessage("doRequest: Setting anti-csrf token");
-                            await AntiCsrfToken.setItem(tok.token, antiCsrfToken);
-                        }
-                    }
-                    const frontToken = response.headers.get("front-token");
-                    if (frontToken) {
-                        logDebugMessage("doRequest: Setting sFrontToken: " + frontToken);
-                        await FrontToken.setItem(frontToken);
-                    }
                     return response;
                 }
             }
```

Comment:
do you also remove legacy sIRTFrontend token when updating to the newer session?

Reply:
We clear it during the access token update.